org-jira.el

```diff
@@ -523,34 +523,43 @@ See `org-default-priority' for more info."
   org-jira-working-dir
   )
 
+(defcustom org-jira-keymap-prefix "C-c"
+  "Org-Jira keymap prefix."
+  :group 'org-jira
+  :type 'string)
+
+
 (defvar org-jira-entry-mode-map
-  (let ((org-jira-map (make-sparse-keymap)))
```

Comment:
What happens in this branch to users who have customized things inside of "org-jira-map"?  Does it still exist as a key map, or does only "org-jira-entry-map" exist?

Reply:
Correct, only `org-jira-entry-mode-map` would exist, but isn't that how it already was?

`org-jira-map` was created as a `let`-scoped variable which is used to contruct a keymap that's then emitted from the `let`-scope for assignment to `org-jira-entry-mode-map`.

Do I have that right?

Reply:
yup!  looks good to me, thank you :)